### ISSUE TITLE
devtools::install_github('royfrancis/pophelper') error

### ISSUE THREAD

**Issue by DanEPlatt** [royfrancis/pophelper#65]
I get the following message:

> devtools::install_github('royfrancis/pophelper')
Error: Failed to install 'unknown package' from GitHub:
  Timeout was reached: [api.github.com] Resolving timed out after 10000 milliseconds

I will offer my own solution: 

1) download the .zip and unzip it.
2) install.packages(<directory>, repos = NULL, type = 'source')

Then library(pophelper) loaded.

**Comment by royfrancis** [comment]
Thanks for reporting this. This is most likely some issue on your end. If you google `Error: Failed to install 'unknown package' from GitHub:`, you will come across some suggestions to fix it. I am not quite sure what the best solution is.

**Comment by DanEPlatt** [comment]
I did google before I tried to report it.  It occurred to me I had the problem before, I think with pop helper, and I think you are the one who first suggested this solution.  So I tried it (again?), and it worked.  

If it happened before, likely it was on a mac.  This time, it was on one of my ubuntu systems.  Googling didn’t show anything particularly useful…

In any case, it did work…  At least there’s a workaround if using the GitHub R tool doesn’t work out.

D
